Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift

```diff
@@ -563,18 +73,21 @@ struct TaggedListItemExtractor: MarkupRewriter {
             }
 
             // Separate all the "- Note:" elements from the other list items.
-            let (noteItems, otherListItems) = unorderedList.listItems.categorize(where: { item in
-                return Aside.Kind.allCases.mapFirst {
-                    item.extractTag($0.rawValue + ": ", dropTag: false)
+            let (noteItems, otherListItems) = unorderedList.listItems.categorize(where: { item -> [BlockMarkup]? in
+                guard let tagName = item.extractTag()?.rawTag.lowercased(),
+                      Aside.Kind.allCases.contains(where: { $0.rawValue.lowercased() == tagName })
+                else {
+                    return nil
                 }
+                return Array(item.blockChildren)
             })
 
             // Add the unordered list with the filtered children first.
             result.append(UnorderedList(otherListItems))
 
             // Then, add the Note asides as siblings after the list they belonged to
             for noteDescription in noteItems {
-                result.append(BlockQuote(Paragraph(noteDescription)))
+                result.append(BlockQuote(noteDescription))
```

Comment:
This isn't part of this change, but why do we need to insert a BlockQuote here? To support multiple paragraphs? Or do all asides have a block quote?

Reply:
Asides are normally written as block quotes. This code basically transforms 
```md
- Note: something noteworthy.

  Second noteworthy paragraph. 
```
into
```md
> Note: something noteworthy.
>
> Second noteworthy paragraph.
```
so that the other code that processes asides only have to consider the block quote syntax.